api/src/pcapi/core/users/models.py

```diff
@@ -278,6 +282,10 @@ def add_underage_beneficiary_role(self) -> None:
     def add_test_role(self) -> None:
         self._add_role(UserRole.TEST)
 
+    def add_anonimized_role(self) -> None:
+        self.roles = []
+        self._add_role(UserRole.ANONYMISED)
```

Comment:
Détail : A priori pas besoin de vérification sur la compatibilité avec les rôles existants, donc on peut simplifier, non ?
```suggestion
        self.roles = [UserRole.ANONYMISED]
```